crypto/evp/digest.c

```diff
@@ -525,14 +524,14 @@ int EVP_Digest(const void *data, size_t count,
     return ret;
 }
 
-int EVP_MD_CTX_set_params(EVP_MD_CTX *ctx, const OSSL_PARAM params[])
+int EVP_MD_CTX_set_params(EVP_MD_CTX *ctx, OSSL_PARAM params[])
 {
     if (ctx->digest != NULL && ctx->digest->set_params != NULL)
         return ctx->digest->set_params(ctx->provctx, params);
     return 0;
 }
 
-int EVP_MD_CTX_get_params(EVP_MD_CTX *ctx, const OSSL_PARAM params[])
+int EVP_MD_CTX_get_params(EVP_MD_CTX *ctx, OSSL_PARAM params[])
```

Comment:
Does this one change back to const?

Reply:
Not the getters. See discussion above

Reply:
No, only the setters get const but I did miss the function immediately above.